### PR TITLE
English Prefix Suffix Search

### PR DESCRIPTION
**Pull request by anAntelope**
So sleep yields sleeping now. And berry yields bearberry.

SharedScreenshot

----
It's Matt. I have to use serena's account for the time being. I have problems with two-factor auth for github login cuz my sim card doesn't work in China. I've requested an account recovering and it takes human verification and time.

### CODE REVIEW

File: CreeDictionary/CreeDictionary/hfstol.py [hunk 1]
```diff
@@ -31,7 +31,7 @@
 def analyze(wordform: str) -> Iterable[Analysis]:
     with write_temporary_file(f"{wordform}\n") as input_file:
         output = check_output(
-            [_hfstol, "-q", _analyzer_path],
+            [_hfstol, "-q", str(_analyzer_path)],
```
Comment (anAntelope) [inline CreeDictionary/CreeDictionary/hfstol.py:34]: windows compatibility

**Comment by anAntelope** [comment]
This should resolve #142

File: CreeDictionary/morphodict/search.py [hunk 2]
```diff
@@ -0,0 +1,840 @@
+"""
```
Comment (eddieantonio) [inline CreeDictionary/morphodict/search.py:1]: Was this commited by mistake? It looks like an old version of `CreeDictionary/API/models.py` that was copy-pasted into `CreeDictionary/morphodict`?

**Review by eddieantonio — CHANGES_REQUESTED** [review]
Uh... please remove `CreeDictionary/morphodict/search.py`???

File: CreeDictionary/morphodict/search.py (hunk 2 above)
Comment (anAntelope) [reply CreeDictionary/morphodict/search.py]: removed, accidentally commited

**Review by eddieantonio — APPROVED** [review]
LGTM!

**Pull request merged**